telegram/_videochat.py

```diff
@@ -42,6 +42,11 @@ class VideoChatStarted(TelegramObject):
 
     __slots__ = ()
 
+    def __init__(self, *, api_kwargs: JSONDict = None) -> None:
+        super().__init__(api_kwargs=api_kwargs)
+
+        self._freeze()
+
 
 class VideoChatEnded(TelegramObject):
```

Comment:
is there a reason for having used a context manager in other modules but leaving `self._freeze()` in this class and the next class in the same module?

Reply:
Yes, that's exactly the reason :) `with self._frozen()` has no advantage here …